docs/src/DDM.md

```diff
@@ -27,8 +31,11 @@ In the code below, we will define parameters for the DDM and create a model obje
 
 The average slope of the information accumulation process. The drift gives information about the speed and direction of the accumulation of information. Typical range: -5 < ν < 5
 
+Across-trial-variability of drift rate. Standard deviation of a normal distribution with mean v describing the distribution of actual drift rates from specific trials. Values different from 0 can predict slow errors. Typical range: 0 < η < 2. Default is 0.
```

Comment:
In order for the code to run correctly across `@example` blocks, you need to add an `@setup` block to define global variables. For example, in the LBA docs, we have:

```
```@setup lba
using SequentialSamplingModels
using SSMPlots 
using Random

ν=[2.75,1.75]
A = .8
k = .5
τ = .3

dist = LBA(;ν, A, k, τ) 
choices,rts = rand(dist, 100)
plot(dist)
histogram!(dist)
```
```
I don't think all of the variables defined there are necessary. So you could reduce the setup to only the necessary global variables if you would like. 

Please add the `@setup` block and test the docs locally with the following procedure:

1. in shell mode, cd to .julia/dev/SequentialSamplingModels
2. in package mode run `activate ./docs`
3. In REPL mode, run `using LiveServer; servedocs()`
4. Using `ctrl` + click, on the hyperlink `http://localhost:8000/ ...` which can be found at the end of the output

This will allow you to verify that the docs render correctly.